pkg/gameserverallocations/controller.go

```diff
@@ -69,13 +71,14 @@ type Controller struct {
 	readyGameServers gameServerCacheEntry
 	// Instead of selecting the top one, controller selects a random one
 	// from the topNGameServerCount of Ready gameservers
-	topNGameServerCount int
-	gameServerSynced    cache.InformerSynced
-	gameServerGetter    getterv1alpha1.GameServersGetter
-	gameServerLister    listerv1alpha1.GameServerLister
-	stop                <-chan struct{}
-	workerqueue         *workerqueue.WorkerQueue
-	recorder            record.EventRecorder
+	topNGameServerCount    int
+	gameServerSynced       cache.InformerSynced
+	gameServerGetter       getterv1alpha1.GameServersGetter
+	gameServerLister       listerv1alpha1.GameServerLister
+	allocationPolicyGetter multiclustergetterv1alpha1.GameServerAllocationPoliciesGetter
```

Comment:
I'm  wondering if it's better to use a Lister so we can take advantage of the its backing cache?

Rather than hit the k8s api every time for configuration I expect won't change very often?

Reply:
Thanks Mark! Exactly, the policies will not change very often and backing cache is more performant.

Reply:
Done